manifests/vhost.pp

```diff
@@ -2398,12 +2404,6 @@
     content => template('apache/vhost/_serversignature.erb'),
```

Comment:
This is just a single line. I'd inline it:
```suggestion
    content => "  ServerSignature Off\n",
```

Reply:
i tried it was failing the acceptance ... feel free to give another shot from your side.

Reply:
https://github.com/puppetlabs/puppetlabs-apache/pull/2467

Reply:
@ekohl  I checked this pipeline and saw that acceptance are passing so accommodating the same changes in this MR as well